backend/src/v5/middleware/dataConverter/inputs/users.js

```diff
@@ -234,6 +239,16 @@ Users.validateSignUpData = async (req, res, next) => {
 	}
 };
 
+Users.validateSsoSignUpData = async (req, res, next) => {
```

Comment:
a lot of duplication with validate `validateSignUpData`

do something like this

````js

const validateSignUpData = (isSSO) => {
try {
		const schema = generateSignUpSchema(isSSO); //Merge this function too.
		req.body = await schema.validate(req.body);
		await next();
	} catch (err) {
		respond(req, res, createResponseCode(templates.invalidArguments, err?.message));
	}
}

Users.validateSignUpData = validateSignUpData(false);
Users.validateSsoSignUpData = validateSignUpData(true);
````